descriptors/ModuleDescriptor-template.json

```diff
@@ -64,7 +64,7 @@
     },
     {
       "id" : "_tenant",
-      "version" : "1.2",
+      "version" : "2.0",
```

Comment:
There is also a suggested changed to make the tenantAPI asynchronous. Please check https://github.com/folio-org/mod-configuration/pull/102/files for more details. It would be good to get it done

Reply:
This is not a mandatory change for this release. So if this is the only change holding up the release, let's not wait. We can incorporate it later